src/main/java/com/tictactoe/Grid.java

```diff
@@ -0,0 +1,121 @@
+package com.tictactoe;
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class Grid {
+    private Board board;
+    private int boardSize;
+    private int cellsPerRow;
+
+    public Grid(Board board) {
+        this.board = board;
+        this.boardSize = board.getBoardSize();
+        this.cellsPerRow = board.getCellsPerRow();
+    }
+
+    private int findMaxStringLengthOfSymbols() {
+        String[] boardArray = this.board.getSymbol();
+        ArrayList<Integer> symbolLength = new ArrayList<>();
+        for (int i = 0; i < boardArray.length; i++) {
+            symbolLength.add(boardArray[i].length());
+        }
+        return Collections.max(symbolLength);
+    }
+
+    private String getPaddedSymbols(String symbol, Integer maxLength) {
+        if (symbol.length() == maxLength) {
+            return symbol;
+        } else {
+            int lengthDifference = maxLength - symbol.length();
+            return this.padLeft(symbol, lengthDifference);
+        }
+    }
+
+    private String padLeft(String symbol, int space) {
```

Comment:
These names are excellent. It is very easy to skim this code and understand what the methods do and where they fit into the main draw method